docs/config.md

```diff
@@ -1800,35 +1802,40 @@ Type: `duration`
 
 ### `Swarm.ResourceMgr`
 
-**EXPERIMENTAL: `Swarm.ResourceMgr` configuration will change in future release**
-
 The [libp2p Network Resource Manager](https://github.com/libp2p/go-libp2p-resource-manager#readme) allows setting limits per a scope,
 and tracking recource usage over time.
 
 #### `Swarm.ResourceMgr.Enabled`
 
-**EXPERIMENTAL: `Swarm.ResourceMgr` is in active development, enable it only if you want to provide maintainers with feedback**
-
-
 Enables the libp2p Network Resource Manager and auguments the default limits
 using user-defined ones in `Swarm.ResourceMgr.Limits` (if present).
 
 Various `*rcmgr_*` metrics can be accessed as the prometheus endpoint at `{Addresses.API}/debug/metrics/prometheus` (default: `http://127.0.0.1:5001/debug/metrics/prometheus`)
 
-Default: `false`
+Default: `true`
 
 Type: `flag`
 
-#### `Swarm.ResourceMgr.Limits`
+#### `Swarm.ResourceMgr.MaxMemory`
```

Comment:
A couple of thoughts:
1. Should we use this type: https://github.com/ipfs/kubo/blob/master/docs/config.md#optionalbytes ?
2. If not, would it make sense to put the unit in the name to make it self documenting like we do in https://github.com/ipfs/kubo/blob/master/docs/config.md#internalbitswapmaxoutstandingbytesperpeer
3. If we are specifying a units, maybe choose MB instead of bytes (and then adjust code accordingly).

That said, item 1 seems best from a user perspective.